cmd/dex-worker/main.go

```diff
@@ -51,6 +51,9 @@ func main() {
 
 	enableClientRegistration := fs.Bool("enable-client-registration", false, "Allow dynamic registration of clients")
 
+	// Client credentials administration
+	forceClientUserCreation := fs.Bool("force-client-user-creation", false, "force create users using admin client-credentials")
```

Comment:
This isn't just about client creation, it's about the entire layer of API access.

Let's do `--api-use-client-credentials`